fritzbox_helper.py

```diff
@@ -25,7 +25,7 @@
 USER_AGENT = "Mozilla/5.0 (U; Windows NT 5.1; rv:5.0) Gecko/20100101 Firefox/5.0"
 
 
-def get_sid(server, password, port=80):
+def get_sid(server, username, password, port=80):
```

Comment:
Set default username to None:
`def get_sid(server, username=None, password, port=80):`